docs/graphql/manual/schema/relay-schema.rst

```diff
@@ -0,0 +1,69 @@
+
+  - It's hard to over-fetch data (which would hurt performance), or under-fetch data (which might cause errors).
+  - Components can only access data they've asked for. This **data masking** prevents implicit data dependency bugs.
+  - Components only re-render when the exact data they're using is updated, preventing unnecessary re-renders.
+
+- **Performance**: The Relay compiler composes your GraphQL fragments into optimized and efficient batches to reduce round-trips to the server. The compiler also applies **transforms** to your queries to remove redundancies and shorten query strings, which reduces upload bytes.
+
+- **Strong typing**: The compiler automatically generates Flow (or TypeScript) types, which you can import into your component for type checking in your code editor and during build time.
+
+To learn more about these and other Relay features, like persisted queries, local state management, passing arguments to fragments, and fetching data as early as possible, check out the `Relay docs <https://relay.dev/docs/en/experimental/a-guided-tour-of-relay>`__.
+
+.. note::
+  For a more detailed breakdown of Relay and its benefits, check out our `deep-dive on Relay <https://hasura.io/blog/deep-dive-into-relay-graphql-client/>`__.
+
+Relay's server spec
+-------------------
+
+To support the above client-side benefits, Relay has a particular server specification.
+
+*Hasura's Relay API sets up this spec for you automatically, so you don't have to implement it manually.*
+
+According to the spec, the server must provide:
+
+- **A mechanism for refetching an object**: The convention is a **Node interface** with a globally unique ``id`` field, as well as a root field called ``node``, which allows fetching data by this ``id``. This is great for performance on the client side, but hard to implement on the server side, since you have to make sure  ``id``'s are globally unique, objects can be re-fetched via their ``id``'s, etc.
+
+- **A description of how to page through connections**: Connections are Relay's way of **standardizing pagination**. They allow us to communicate more info between the client and the server, such as cursors and page info, so that we can paginate in a predictable pattern. On the client side, Relay saves you a ton of work by keeping track of the moving parts of pagination and merging results automatically.
+
+.. note::
+  Check out this `example repo <https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/react-relay>`__ to see how to set up pagination with Hasura and Relay.
+
+Limitations
+-----------
+
+At this time, Hasura's Relay implementation doesn't cover remote schemas and actions, and doesn't support persisted queries. It supports only Postgres tables with a primary key defined, and custom SQL functions whose returning table has a primary key defined.
```

Comment:
```suggestion
At this time, Hasura's Relay support only Postgres tables with a primary key defined, and custom SQL functions whose returning table has a primary key defined.

Persisted queries will be supported soon.
```

The above is relay specific. After this I will add a note admonition which says:

```
At this time, Hasura's Relay schema doesn't expose remote schemas and actions. This will be fixed in future releases.
```

Reply:
I'm getting the same html with `.. note::` and `.. admonition::` -- what's the difference? @marionschleifer do you know?